deflate_medium.c

```diff
@@ -165,6 +165,8 @@ Z_INTERNAL block_state deflate_medium(deflate_state *s, int flush) {
     ALIGNED_(16) struct match current_match;
                  struct match next_match;
 
+    int level = s->level;
```

Comment:
I still think it should be `early_exit = s->level >= 5` even though the actual calculation itself might not give much advantage to precalculate it, the `early_exit` variable describes why there is `level >= 5` break out in those places.

Reply:
Oh, right. I actually forgot to do that after I cleaned up a bunch of testing-code.
It makes no discernable difference in speed (since it makes the if compare it to 1 instead of 5), but it is clearer.

Reply:
Pushed the change now. It does require a negation in the if statements too though.